extensions/notebook/src/test/common/port.test.ts

```diff
@@ -53,7 +53,7 @@ describe('Ports', () => {
 				options.maxRetriesPerStartPort = 1;
 				options.totalRetryLoops = 50;
```

Comment:
I still think it's worth reducing the retry count and timeout - they should never actually hit that high except in error scenarios and making the test hang for hours in that case isn't great.